docs/Contributing/Releasing-Fleet.md

```diff
@@ -2,19 +2,33 @@
 
 ## Release process
 
-This section outlines the release process at Fleet.
+This section outlines the release process at Fleet after all EMs have certified that they are ready for release. 
 
-The current release cadence is once every three weeks and concentrates around Wednesdays.
+The current release cadence is once every three weeks and concentrates around Wednesdays. Patch versions are released as needed.
```

Comment:
Should this be updated to Tuesdays w/ the first day of freeze being Tuesday now?

Reply:
Yes, it should, thanks! Will update.